src/demo-app/progress-circle/progress-circle-demo.scss

```diff
@@ -0,0 +1,9 @@
+.progress-circle-container {
```

Comment:
I'm going to do this for the existing demos soon, but could you prefix the class with `demo-` ?